### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinegeffen**
General typos and formatting.

### CODE REVIEW

**Review by larkox — APPROVED** [review]
LGTM. Thanks!

**Review by hanzei — APPROVED** [review]

File: docs/installation/zoom-configuration/README.md [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
 # Zoom Configuration
 
-Zoom supports two authentication methods for your users to connect Mattermost and Zoom. Choose between using OAuth and JWT/Password based authentication:
+Zoom supports two authentication methods for users to connect Mattermost and Zoom: OAuth or JWT/Password based authentication.
 
-1. [Oauth: ](zoom-setup-oauth.md)
-   * Each user will need to connect their Zoom account with their Mattermost account before they can use the integration.  When they try to create a meeting for the first time, they will receive a message to connect their account, and will need to click "Approve" on a pop-up web page.
-   * Users can connect their MM/Zoom accounts even if their emails do not match
+1. [Oauth](zoom-setup-oauth.md)
+   * Users need to connect their Zoom account with their Mattermost account before they can use the integration. When they try to create a meeting for the first time, they'll receive a message to connect their account. They'l need to select **Approve** on a pop-up web page, in order to begin the meeting.
```
Comment (larkox) [inline docs/installation/zoom-configuration/README.md:12]:
```suggestion
   * Users need to connect their Zoom account with their Mattermost account before they can use the integration. When they try to create a meeting for the first time, they'll receive a message to connect their account. They'll need to select **Approve** on a pop-up web page, in order to begin the meeting.
```

File: docs/installation/zoom-configuration/README.md [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
+   * Users can connect their Mattermost/Zoom accounts even if their email addresses do not match.
 2. [JWT/Password](zoom-setup-jwt.md)
-   * Users don't have to connect their account to use the integration which makes it easy to get started.  
-   * Uses JWT to pass security tokens.  This may not be sufficiently secure for some customers.
-
-
-
+   * Users don't have to connect their account to use the integration which makes it easy to get started.
+   * Uses JWT to pass security tokens. This may not be sufficiently secure for some customers.
```
Comment (larkox) [inline docs/installation/zoom-configuration/README.md:16]: I would add something like:
* The users must have the same email registered both in Zoom and Mattermost.

**Review by larkox — CHANGES_REQUESTED** [review]
One small typo, and one addition.

**Review by larkox — APPROVED** [review]
LGTM

**Pull request merged**